etl/steps/data/garden/technology/2023-03-06/mobile_money.meta.yml

```diff
@@ -0,0 +1,33 @@
+dataset:
+  namespace: technology
+  short_name: mobile_money
+  title: Global Mobile Money Dataset (GSMA, 2022)
+  description: |-
```

Comment:
Metadata is inherited as `snapshot -> meadow -> garden` so you don't need to repeat it at each step. You can easily remove garden's `mobile_money.meta.yml` and only keep this in meadow's

```
dataset:
  sources:
    - name: GSMA (2022)
      url: https://www.gsma.com/mobilemoneymetrics/#global
      source_data_url: https://www.gsma.com/mobilemoneymetrics/wp-content/uploads/2022/05/Global_Mobile_Money_Dataset_new.xlsx
      date_accessed: '2023-03-06'
      publication_date: '2022-05-01'
      publication_year: 2022
      published_by: GSMA
tables:
  mobile_money:
    variables:
      active_accounts_90d:
        title: 'Active accounts (90 day)'
        short_unit: ''
        unit: 'active accounts'
        description: "The number of customer accounts that have been used to perform at least one P2P payment, bill payment, bulk payment, cash in to account, cash out from account, merchant payments, international remittances or airtime top up from account during at least 90 days prior to end of the indicated months. Balance inquiries, PIN resets, and other transactions that do not involve the movement of value do not qualify a customer account as active."
        display:
          numDecimalPlaces: 0
```